### PR TITLE
feature/unsubscribe from given subscription

### PR DESCRIPTION
**Pull request by JGiter**
- `unsubscribeFromMessage` will allow to specify subscription id
- `unsubscribeFromMessage` and `subscribeToMessages` shortened to `unsubscribeFrom` and `subscribeTo`

### CODE REVIEW

**Review by jrhender — CHANGES_REQUESTED** [review]
Code looks good. But I believe the commit message should include a `BREAKING CHANGE:` footer as the API is being changed: https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#footer

File: src/iam.ts [hunk 1]
```diff
@@ -1418,26 +1418,28 @@ export class IAM extends IAMBase {
...
+    for await (const msg of subscription) {
       const decodedMessage = this._jsonCodec?.decode(msg.data) as IMessage;
       messageHandler(decodedMessage);
     }
   }
 
-  async unsubscribeFromMessages() {
-    if (this._exchangeSubscription) {
-      this._exchangeSubscription.unsubscribe();
+  async unsubscribeFrom(subject: string) {
```
Comment (kim-energyweb) [inline src/iam.ts:1439]: How can I unsubscribe to subscription 3 if I have below set of codes:
```
this.iam.subscribeTo({ messageHandler: this.myHandler1 }); // subscription 1
this.iam.subscribeTo({ messageHandler: this.myHandler2 }); // subscription 2
this.iam.subscribeTo({ messageHandler: this.myHandler3 }); // subscription 3
```

Comment (JGiter) [reply src/iam.ts]: Fixed. Subscriptions will be identified by id

**Review by kim-energyweb — APPROVED** [review]
LGTM

**Review by jrhender — APPROVED** [review]

**Pull request merged**